Microsoft.Toolkit.Uwp.UI.Controls/BladeControl/BladeControl.cs

```diff
@@ -58,6 +59,8 @@ public BladeControl()
         {
             DefaultStyleKey = typeof(BladeControl);
             Blades = new ObservableCollection<Blade>();
+
+            Window.Current.SizeChanged += WindowSizeChanged;
```

Comment:
Listen to the SizeChanged event of the control itself rather than the entire window